example/example.dart

```diff
@@ -107,5 +108,71 @@ void main() async {
   print('Access Token: ${credentials.accessToken}');
   print('Credentials Expired: ${credentials.isExpired}');
 
+  print('Would you like to access user specific data? [y/n]');
```

Comment:
Awesome work getting this CLI set up, it'll make testing this package so much easier!

My only concern is that including all of these changes directly in `example.dart` makes the file really big. It's tough because right now this file is providing both a gentle intro to the package on the [pub.dev Example tab](https://pub.dev/packages/spotify#-example-tab-) and a testing ground for changes, which are pretty different use cases.

One way you could address that would be to break off the authorization code into a separate file like `example_auth.dart`. I'm curious to hear other people's thoughts though!

Reply:
Yeah, having examples needing oauth in their own file is probably a good idea.

Reply:
I agree, sounds more clean.